tool/tsh/kube.go

```diff
@@ -611,6 +634,7 @@ func newKubeLSCommand(parent *kingpin.CmdClause) *kubeLSCommand {
 	}
 	c.Flag("search", searchHelp).StringVar(&c.searchKeywords)
 	c.Flag("query", queryHelp).StringVar(&c.predicateExpr)
+	c.Flag("format", "Format output (text, json, yaml)").Short('f').Default(teleport.Text).StringVar(&c.format)
```

Comment:
In another command we have: `c.Flag("format", "Format output (text, json, yaml)").Short('f').Default(teleport.Text).EnumVar(&c.format, defaultFormats...)`. I think we can use `EnumVar` everywhere consistently?